content/02.introduction.md

```diff
@@ -1,2 +1,45 @@
+The first known cases were centralized in Wuhan City within the Hubei province of China, but the diease subsequently began to spread rapidly beyond Wuhan within China and around the world. 
+At the time of the first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4], 282 confirmed cases had been identified, primarily in China, but also the first 1 to 2 cases in each of Thailand, Japan, and the Republic of Korea.
+One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@urL:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf?sfvrsn=8b671ce5_2].
+On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf?sfvrsn=1ba62e57_10].
+By WHO Situation Report 61, released on March 20, 2020, 266,073 confirmed cases had been reported worldwide, with cases on every continent except Antarctica [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200321-sitrep-61-covid-19.pdf?sfvrsn=6aa18912_2].
+At this time, over 11,000 deaths had been reported worldwide.
+
+[Note: Maybe add a graph here, update as new reports come out.]
+
+COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
+SARS-CoV-2 is a coronavirus, a type of RNA virus known to cause respitatory infections in humans and other species.
+Other well-known coronaviruses include those associated with previous infectious diseases of global concern, including Severe Accute Repiratory Syndrome (SARS) and Middle East respiratory syndrome (MERS); however, some coronaviruses are associated with less virulent illnesses including the common cold.
+The SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4].
+As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, information about the transmission and life cycle of the virus as well as the diagnosis and treatment of the disease is emerging rapidly.
+In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
+This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.
+
+### Coronaviruses: What are they, and what do we know about SARS-CoV-19?
+
+Coronaviruses are RNA viruses that...
```

Comment:
Do you want to include mechanisms of entry into cells here? Or would it go below? I don't think it needs to be written but I'm not sure where it would go. A lot of folks are thinking about ACE2, so having some discussion somewhere is probably good.

Reply:
Should be addressed in commit 40b872b below